### DISCUSSION TITLE
cy.task('sqlServer:execute') failed with the following error:  > Failed to connect to

### DISCUSSION THREAD

**Discussion by leolpc21** [cypress-io/cypress#25963] [Questions and Help]
Bom dia pessoal, tudo bem?
Estou tendo problema para fazer a conexão com o SQL Server, teriam como me ajudar?

Segue minhas configurações:

package.json
![image](https://user-images.githubusercontent.com/43275999/221588492-092cdba1-abc1-4214-b0da-79be3c081f52.png)

cypress.config.js
![image](https://user-images.githubusercontent.com/43275999/221588107-aa0518d7-349e-4f3a-9a95-bd530e65129d.png)

support/e2e.js
![image](https://user-images.githubusercontent.com/43275999/221588366-a0d06704-8665-47da-8f60-826b9275dc03.png)

cy.js
![image](https://user-images.githubusercontent.com/43275999/221589478-52eccebe-a6c4-410a-94a9-3ba0795a076a.png)

Erro
![image](https://user-images.githubusercontent.com/43275999/221588967-74a20ad6-05b3-4f45-bf1a-bf7f15d5db47.png)

**Comment by zeapaez** [comment]
Same issue, @leolpc21  were you able to solve it??

**Reply by leolpc21** [reply]
Não tive nenhum retorno e sem resolução.
